roles/foreman_provisioning/tasks/configure_centos_9.yml

```diff
@@ -0,0 +1,27 @@
+- name: "Ensure CentOS Stream 9"
+  theforeman.foreman.operatingsystem:
```

Comment:
When you try to run this, it will complain you did not define `password_hash`.

Congratulations, you just found a bug in the Ansible modules! :)

I'll go fix it, but in the meantime you can add `password_hash: SHA512`

Reply:
https://github.com/theforeman/foreman-ansible-modules/pull/1779